students/yushus/session04/mailroom2.py

```diff
@@ -0,0 +1,93 @@
+import os
+import random
+import sys
+
+donor_db = {"William Gates, III": [653772.32, 12.17],
+            "Jeff Bezos": [877.33],
+            "Paul Allen": [663.23, 43.87, 1.32],
+            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0],
+            "Elon Musk": [250000, 15000]}
+
+prompt = "\n".join(("Welcome to mailroom bot!",
+          "Please choose from below options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Send letters to all donors",
+          "4 - quit",
+          ">>> "))
+
+def print_donor():
+    donors = donor_db.keys()
+    donors_list = "\n".join(donors)
+    print(donors_list)
+
+def format_letter():
```

Comment:
Minor: Looks like this function returns a fixed string (a formatting string), so not sure if it needs to be a function. The `letter` variable could have been declared at the global level (though, with some other/proper name), and just use the global variable in `print_email()`. Well, using global variables is kind of frowned upon, and in that case, you could receive `name` and `amount` as parameters to this function and return the actually constructed strings, not just a format string. Just FYI, and no need to change this.